client/src/components/StreamField/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore, applyMiddleware } from 'redux';
```

Comment:
just curious, did you ever tried Appollo client or mobx?

Reply:
No, I didn’t. Thank you for pointing them out, Mobx especially seems very interesting! I will probably use it in other projects.
As for Wagtail, the choice was already settled with the other React-based tools that we have that all use Redux, especially Draftail.

Reply:
OK thanks. I started with redux back in 2016, but later moved to mobx and recently appollo!